python/snippets/stt_streaming_recognize_gender_identification.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+import sys
+sys.path.append("..")
+
+from tinkoff.cloud.stt.v1 import stt_pb2_grpc, stt_pb2
+from auth import authorization_metadata
+import grpc
+import os
+import wave
+
+endpoint = os.environ.get("VOICEKIT_ENDPOINT") or "stt.tinkoff.ru:443"
+api_key = os.environ["VOICEKIT_API_KEY"]
+secret_key = os.environ["VOICEKIT_SECRET_KEY"]
+
+def build_first_request(sample_rate_hertz, num_channels):
+    request = stt_pb2.StreamingRecognizeRequest()
+    request.streaming_config.config.encoding = stt_pb2.AudioEncoding.LINEAR16
+    request.streaming_config.config.sample_rate_hertz = sample_rate_hertz
+    request.streaming_config.config.num_channels = num_channels
+    request.streaming_config.config.enable_gender_identification = True
+    return request
+
+def generate_requests():
+    try:
+        with wave.open("../../audio/stierlitz.wav") as f:
+            yield build_first_request(f.getframerate(), f.getnchannels())
+            frame_samples = f.getframerate()//10 # Send 100ms at a time
+            for data in iter(lambda:f.readframes(frame_samples), b''):
+                request = stt_pb2.StreamingRecognizeRequest()
+                request.audio_content = data
+                yield request
+    except Exception as e:
+        print("Got exception in generate_requests", e)
```

Comment:
a think its too much, because it didn't carries additional information about error, and finally i will see big stack trace anyway.